### PR TITLE
Mg/homologouscollapse

### PR DESCRIPTION
**Pull request by mari2895**
## PR Checklist

- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [ ] Format your changes by calling `scripts/bash/format.sh`.
- [ ] Explain what you did.

### CODE REVIEW

**Review by Yurlungur — CHANGES_REQUESTED** [review]
Overall looks good! A few comments below. Can you also format the code by running that format script and committing the changed files?

File: src/phoebus_utils/variables.hpp [hunk 1]
```diff
@@ -78,6 +78,11 @@ constexpr char fail[] = "fail";
 constexpr char Gcov[] = "Gcov";
 constexpr char Gye[] = "Gye";
 constexpr char c2p_mu[] = "c2p_mu";
+constexpr char tau[] = "light_bulb_tau";
+constexpr char GcovHeat[] = "GcovHeat";
+constexpr char GcovCool[] = "GcovCool";
+constexpr char compweight[] = "compweight";
```
[the remark this answers is not in the document]
Comment (mari2895) [reply src/phoebus_utils/variables.hpp:84]: I can delete these, don't think we need it anymore. What do you think?

File: src/radiation/radiation.cpp [hunk 2]
```diff
@@ -116,10 +117,26 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
     const bool do_liebendorfer =
         pin->GetOrAddBoolean("radiation", "do_liebendorfer", false);
     const bool do_lightbulb = pin->GetOrAddBoolean("radiation", "do_lightbulb", false);
-    const Real lum = pin->GetOrAddReal("radiation", "lum", 4.0e52);
+    const Real lum = pin->GetOrAddReal("radiation", "lum", 4.0);
```
[the remark this answers is not in the document]
Comment (mari2895) [reply src/radiation/radiation.cpp:120]: I think this is an old file, mine has 4.0.

File: src/phoebus_utils/variables.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (Yurlungur) [reply src/phoebus_utils/variables.hpp:84]: I think maybe we still need tau. But I think the others can go away.

File: src/radiation/radiation.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (Yurlungur) [reply src/radiation/radiation.cpp:120]: The diff is showing it changed to 4.0. Nevertheless, just curious. What are the units here?

**Review by Yurlungur — APPROVED** [review]

**Pull request merged**